src/common/components/form/FormErrorNotification.tsx

```diff
@@ -0,0 +1,61 @@
+import { useFormikContext } from 'formik';
+import { Notification } from 'hds-react';
+import * as React from 'react';
+
+import { useTranslation } from '../../../i18n';
+import styles from './errorMessage.module.scss';
+import { doFocus as focusToFirstError } from './FocusToFirstError';
+
+const FormErrorNotification: React.FC<{
+  errors: string[];
+  visible: boolean;
+}> = ({ errors, visible }) => {
+  const { t } = useTranslation();
+  const { submitCount } = useFormikContext();
+  const containerRef = React.useRef<HTMLDivElement>(null);
+  const tabUsed = React.useRef<boolean>(false);
+
+  React.useEffect(() => {
+    containerRef.current?.focus();
+    tabUsed.current = false;
+  }, [submitCount]);
+
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLDivElement>) => {
+    if (e.key === 'Tab' && !e.shiftKey) {
+      const focused = focusToFirstError();
+      if (focused) {
+        e.preventDefault();
+      }
+    }
+  };
+
+  // Focus first error element after tabbing from notification
+  const handleOnBlur = () => {
+    if (tabUsed.current) {
+      focusToFirstError();
+    }
+  };
+
+  return visible ? (
+    <div
+      className={styles.errorNotificationContainer}
+      role="alert"
+      tabIndex={-1}
+      ref={containerRef}
+      key={submitCount}
+      onBlur={handleOnBlur}
+      onKeyDown={handleKeyDown}
+    >
+      <Notification label={t('form:errorNotification.title')} type="error">
+        {t('form:errorNotification.listTitle')}
+        <ul>
+          {errors.map((error) => (
+            <li key={error}>{t(error)}</li>
```

Comment:
In previous project where I have been, there is almost identical solution, expect for that errors were clickable links, which would move the focus to that invalid input. I cant remember did it use anchor links or focus(). No need to done here now, but just a suggestion. That form was also approved by accessibility expert.

Reply:
Sounds like nice to have feature but not very important 🤔